### DISCUSSION TITLE
[Desktop] ScrollViewer does not support Zoom

### DISCUSSION THREAD

**Discussion by kucint** [unoplatform/uno#16717] [Q&A] [answered]
Hey, it seems that `ScrollViewer` does not support zoom yet on Desktop `TargetFrameworks`.

![image](https://github.com/unoplatform/uno/assets/54866088/922eefdd-b7c0-4853-bc41-dbf284154614)

`ZoomFactor` method seems to be not implemented.

Any chance to add zoom support in a reasonable time? This feature is crucial for the product my company is currently developing.

**Comment by limeniye** [comment, marked as answer]
Looks as the following issue: https://github.com/unoplatform/uno/issues/14350

**Reply by kucint** [reply]
@limeniye Indeed! Unfortunately no action here since 1y. Can this issue be given more attention?

**Reply by limeniye** [reply]
Let's react it via emoji
